e2e/page-objects/mapPage.ts

```
@@ -167,8 +172,8 @@ export class MapCanvas extends HelperBase {
 
     return {
       zoomInteger: Math.trunc(zoom),
-      latitudeInteger: Math.trunc(latitude),
-      longitudeInteger: Math.trunc(longitude),
+      latitudeInteger: Math.trunc(latitude) === -0 ? 0 : Math.trunc(latitude),
```

Biome (1.9.4)

[error] Do not use the === operator to compare against -0.

Safe fix: Replace -0 with 0


(lint/suspicious/noCompareNegZero)